.eslintrc.json

```diff
@@ -17,7 +17,7 @@
         "ecmaFeatures": {
             "jsx": true
         },
-        "ecmaVersion": 2018,
+        "ecmaVersion": 2020,
```

Comment:
I've used some newer language features, specifically optional chaining, so this needed updating:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining